taskmanager/src/Infrastructure/Adapters/Outgoing/PDO/TaskRepository.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Infrastructure\Adapters\Outgoing\PDO;
+
+use App\Domain\Task\Status;
+use App\Domain\Task\Task;
+use App\Domain\User\User;
+
+class TaskRepository implements \App\Domain\Ports\Outgoing\TaskRepository
+{
+    private $db;
+
+    public function __construct(\PDO $pdo)
+    {
+        $this->db = $pdo;
+    }
+
+    public function getAll()
+    {
+        $sth = $this->db->prepare('SELECT * FROM task');
+        $sth->execute();
+        $results = $sth->fetchAll();
+
+        foreach ($results as $item) {
+            $tasks[] = [
```

Comment:
it should be a `Task` object instead, should not be?

Reply:
I was not sure before (dependency flow) but yes - should be.